.gitignore

```diff
@@ -5,3 +5,4 @@ phpunit.xml
 composer.phar
 package.tar
 /packages.json
+/.phpunit
```

Comment:
@andrerom I'm talking about the logic to locate PHPunit in our CI setup, which is running the testsuite both for the fullstack repo and for each component. If it is in the require_dev section of each component, it means we have a different PHPunit location for each component